lib/main.dart

```diff
@@ -57,10 +57,18 @@ class App extends StatelessWidget {
       child: _WithSelectedNetwork(
         initialNetwork: initialNetwork,
         child: _WithTermsAndConditionAcceptance(
-            child: MaterialApp(
-          routes: appRoutes,
-          theme: concordiumTheme(),
-        )),
+          child: MultiBlocProvider(
```

Comment:
The intention was to provide an anchor for adding future providers that should result in app-wide changes, e.g., changing the language. A MultiBlocProvider allows you to specify the providers (theme, language, etc.) in a list, rather than having to nest them.